ui/packages/catalog-ui-search/src/main/webapp/react-component/location/base.line.js

```diff
@@ -20,32 +35,101 @@ class BaseLine extends React.Component {
   render() {
     const props = this.props
     const { label, cursor, geometryKey, unitKey, widthKey } = props
-
     return (
-      <div className="input-location">
-        <TextField
-          label={label}
-          value={this.state.value}
-          onChange={value => {
-            this.setState({ value })
-            const fn = cursor(geometryKey)
-            try {
-              fn(JSON.parse(value))
-            } catch (e) {}
-          }}
-        />
-        <Units value={props[unitKey]} onChange={cursor(unitKey)}>
+      <React.Fragment>
+        <div className="input-location">
           <TextField
-            type="number"
-            label="Buffer width"
-            min={0.000001}
-            value={props[widthKey]}
-            onChange={cursor(widthKey)}
+            label={label}
+            value={this.state.value}
+            onChange={value => {
+              this.setState({ value })
+              const fn = cursor(geometryKey)
+              try {
+                fn(JSON.parse(value))
+              } catch (e) {}
+            }}
+            onBlur={() => this.isValidInput(this.state.value)}
```

Comment:
I don't think you need the arrow function since you bound the function in the constructor
I believe you should be able to do this instead:
```suggestion
            onBlur={this.isValidInput(this.state.value)}
```